### ISSUE TITLE
broken ebuilds

### ISSUE THREAD

**Issue by Alessandro-Barbieri** [ros/ros-overlay#31]
Please fix this

```
/var/lib/layman/ros-overlay/ros-kinetic/image_geometry/image_geometry-1.12.4.ebuild: line 9: image_geometry: command not found
/var/lib/layman/ros-overlay/ros-kinetic/imu_complementary_filter/imu_complementary_filter-1.1.4.ebuild: line 6: a: command not found
/var/lib/layman/ros-overlay/ros-lunar/image_geometry/image_geometry-1.12.4.ebuild: line 9: image_geometry: command not found
/var/lib/layman/ros-overlay/ros-lunar/imu_complementary_filter/imu_complementary_filter-1.1.5.ebuild: line 6: a: command not found
```

**Comment by allenh1** [comment]
So the ebuilds will emerge, but there is still an issue: the backtick '`' is somehow trying to run a subshell in the description of the ebuilds. I am going to fix the description field parsing in the near future.

They are also over 80 columns in length.

**Comment by allenh1** [comment]
Fixed 60710da.
